src/reformat.c

```diff
@@ -115,11 +115,11 @@ avifBool avifGetYUVColorSpaceInfo(const avifImage * image, avifYUVColorSpaceInfo
         return AVIF_FALSE;
     }
 
+    // Removing 400 here would break backward behavior but would respect the spec.
```

Comment:
But this is checked for later? What path does this break?

Reply:
Nope, I chose the latest in the pipeline before calling the encoder to check for that. I confirm though that with a recent libaom and commenting out the return in src/write.c that I just added, the losslesstest fails.